### ISSUE TITLE
PA Review: Notes

### ISSUE THREAD

**Issue by milesmcleod** [milesmcleod/purplenote#4]
Link: [Purplenote Heroku](https://purplenote.herokuapp.com)

- [x] User can create notes
- [x] User can  read notes
- [x] User can  update notes
- [x] User can sort notes according to multiple preferences
- [x] User can delete notes
- [x] Adequate styling
- [x] Smooth, bug-free navigation
- [x] Adequate and appropriate seeds to demonstrate the feature

**Comment by AAlfarho** [comment]
Great job on the notes, the sorting, attention to the detail on the animations and the autosave are amazing.  Just a couple of details

+ When creating a new note, let's enable the `esc` functionality to cancel the creation.
+ Add some margin or padding on your content's _textarea_, when the content is too long the component will expand to the bottom of the page.
+ Is the favorite/star functionality working?
+ When changing a notebook on the note, it is not possible to close the dropwon by hitting `esc` or clicking aywhere outside of it, lets change this. The dropdown remains visible even after clicking on a different note.
+ When selecting a note and hitting fullscreen/done, this action is triggering AJAX requests for the same note, is this necessary?

**Comment by milesmcleod** [comment]
Yo so I've gone through and enabled users to click out of dropdowns and use the escape key for that and for fullscreen. The star/bookmark functionality isn't functional yet; I believe this isn't in the MVPs, so it'll be on the later side of what I implement, but I think I will definitely get to it. I've also added a margin to the bottom of the text area. I'll push it up to Heroku in a few minutes. As for the AJAX firing each time I come in and out of fullscreen, I think I fixed it. Now it only saves on a manual save, an autosave, or when the user clicks away from the note. I need to play around with it a bit more to be sure. I do have a pretty in-the-weeds question to talk to you about my fullscreen functionality that isn't pressing, for when we meet.
